understanding/22/accessible-authentication-enhanced.html

```diff
@@ -12,6 +12,16 @@ <h1>Understanding Accessible Authentication (Enhanced)</h1>
         <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section> 
+   
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make logins possible with less mental effort.</dd>
+            <dt>What to do</dt><dd>Don’t make people recognize objects or recall personal information to login.</dd>
+            <dt>Why it's important</dt><dd>Some people with cognitive disabilities cannot identify objects.</dd>
```

Comment:
WE know that this SC builds on 3.3.8, but taken in isolation the WHY for this SC only addresses part of the WHAT and appears incomplete. Maybe something like: "Some people with cognitive disabilities cannot memorize a username and password, or type them, nor identify objects."

Reply:
yes, I'm sure I made a similar point previously. the "enhanced" ones should be additive here, not just point out the delta to the "minimum"

Reply:
I  want to set the context that for almost all these SCs, the language of the first two lines is not being changed. It's being repositioned, and a third line added.
So, while I think this round gives us a chance to revisit existing wording -- _especially_ in a case like this where the wording _has_ been changed for this PR -- I want to emphasize that the third line (and overall repositioning) is the focus of the PR (and I get that the repositioning may motivate some of these comments).

Just trying to manage expectations :)

Reply:
Remember that they're going to pass AA with those other details.
I'm not convinced there's a sufficient win by making these cumulative. They become harder to 'spot' the difference, placing more demand on the user to do comparisons.  But I'm also willing to explore. :)
I suspect the same overall response from the WG (as per Patrick's and others earlier comments) will apply here: it's more valuable to emphasize the addition or otherwise make the difference obviously, where possible.

Reply:
then make it explicit with something like "in addition to its 'Minimum' counterpart, ..."

unfortunately, the way our 'Enhanced' SCs themselves have been worded *is* cumulative, so the brief overview should really reflect that. yes, *in practice* the whole SC only brings an extra aspect to the table, but the SC itself is written to include the AA norms as well

Reply:
```suggestion
            <dt>Why it's important</dt><dd>Some people with cognitive disabilities are excluded by cognitive function test, including object identification.</dd>
```

Reply:
I'd like to keep the wording simple and direct. maybe:
> Some people with cognitive disabilities can't do puzzles and tests, and can't identify objects.

Reply:
```suggestion
            <dt>Why it's important</dt><dd>Some people with cognitive disabilities can't do puzzles and tests, or identify objects and media.</dd>
```